Dockerfiles/admin/Dockerfile

```diff
@@ -134,7 +129,6 @@ RUN { \
  && chown -R "${OPENCAST_USER}:${OPENCAST_GROUP}" "${OPENCAST_HOME}" "${OPENCAST_DATA}" \
   \
   # Cleanup
- && apk del --no-cache nodejs \
```

Comment:
The `nodejs` package is now part of the `.build-deps` group, which is removed one line below.